subtests/docker_cli/systemd/p4321.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=docker-autotest systemd subtest. If you see this file on a live system it means something has failed very badly
+After=docker-latest.service
```

Comment:
Is this functionality only ever to be supported in whatever docker-latest is?

Maybe supply two unit files, one for ``docker``, another for ``docker-latest``.  Then add a config option that names which to use.  That way it's up to the tester which to set.